packages/lisk-p2p/test/unit/errors.ts

```diff
@@ -83,4 +87,74 @@ describe('errors', () => {
 			});
 		});
 	});
+
+	describe('#RPCGetPeersFailed', () => {
+		let rpcGetPeersFailed: RPCGetPeersFailedError;
+		const peerId = '127.0.0.1:5001';
+		const defaultMessage = `Error when fetching peerlist of peer with peer id ${peerId}`;
+		const defaultError = new Error('Peer not available');
+
+		beforeEach(() => {
+			rpcGetPeersFailed = new RPCGetPeersFailedError(
+				defaultMessage,
+				defaultError,
+				peerId,
+			);
+			return Promise.resolve();
+		});
+
+		describe('should create an error object', () => {
+			it('should create a new instance of RPCGetPeersFailed', () => {
+				return expect(rpcGetPeersFailed)
+					.to.be.an('object')
+					.and.be.instanceof(RPCGetPeersFailedError);
+			});
+
+			it('should set error name to `RPCGetPeersFailed`', () => {
+				return expect(rpcGetPeersFailed.name).to.eql('RPCGetPeersFailed');
+			});
+		});
+
+		describe('should set error object properties', () => {
+			beforeEach(() => {
+				rpcGetPeersFailed = new RPCGetPeersFailedError(
+					defaultMessage,
+					defaultError,
+					peerId,
+				);
+				return Promise.resolve();
+			});
+
+			it('should set error property info when passed as an argument', () => {
+				return expect(rpcGetPeersFailed)
+					.and.to.have.property('peerId')
+					.which.is.eql(peerId);
+			});
+
+			it('should set error property info when passed as an argument', () => {
```

Comment:
These two test cases look the same (including the assertions) `should set error property info when passed as an argument`.

Reply:
Oh, I forgot to change the messages here :P

Reply:
Initially, I added the whole VError cause and info object and removed it later but forgot to remove from the tests